.github/workflows/build-and-push-cronjob-image.yaml

```
@@ -47,6 +47,7 @@ jobs:
           image: ${{ matrix.image }}
           context: ${{ env.path }}
           oci: true
+          tags: latest "${ GITHUB_SHA::7 }"
```

I think it's either `${GITHUB_SHA::7}` (without the whitespaces) or `${{ GITHUB_SHA::7 }}` (but that's I think syntax for [expressions](https://docs.github.com/en/actions/learn-github-actions/expressions) or [contexts](https://docs.github.com/en/actions/learn-github-actions/contexts)).
And I'd probably try to remove the quotes as well.
```suggestion
          tags: latest ${GITHUB_SHA::7}
```